students/jakielczyk_miroslaw/lesson_05_lists/age_calculator.py

```diff
@@ -0,0 +1,21 @@
+def calculate_average_age(list_of_ages):
+    adult_numbers = 0
+    adult_age_summary = 0
+    child_numbers = 0
+    for element in list_of_ages:
```

Comment:
it's fine, but it's also a good place to exercise list comprhension e.g.:
`adults_age = [age for age in age_of_people if age >= 18]`